galaxy/api/v2/views/collection.py

```diff
@@ -165,6 +166,21 @@ def _check_role_name_conflict(self, ns, name):
             'or rename the role via the meta/main.yml role_name attribute'
         )
 
+    def _check_multi_repo_name_conflict(self, ns, name):
+        multi_content_repos = models.Repository.objects.filter(
+            format='multi',
+            name__iexact=name,
+        )
```

Comment:
it doesn't look like this filters on the namespace at all

Reply:
Updated